src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private final ArrayList<Task> taskList;
+    
+    public TaskList(ArrayList<Task> tasks) {
+        taskList = tasks;
+    }
+    
+    public Task get(int index) {
+        return taskList.get(index);
+    }
+
+    public ArrayList<Task> getTaskList() {
+        return this.taskList;
+    }
+    
+    public int size() {
```

Comment:
```suggestion
    public int getTaskCount() {
```
Would it be better to rename this method to sound more like a verb? It may be more easily understood when calling this method from another class.